<launch>

  <node pkg="robotsocket" type="harvestingRobotClient" name="harvestingRobotClient" output="screen"/>
  
</launch>